pkg/performanceprofile/controller/performanceprofile_controller.go

```diff
@@ -387,21 +393,32 @@ func (r *PerformanceProfileReconciler) Reconcile(ctx context.Context, req ctrl.R
 		return reconcile.Result{RequeueAfter: 10 * time.Second}, nil
 	}
 
-	klog.Info("Reconciling PerformanceProfile")
-	// Fetch the PerformanceProfile instance
-	instance := &performancev2.PerformanceProfile{}
-	err = r.Get(ctx, req.NamespacedName, instance)
+	klog.InfoS("Reconciling", "reqNamespace", req.NamespacedName)
+	defer klog.InfoS("Exit Reconciling", "reqNamespace", req.NamespacedName)
```

Comment:
again once we're confident enough we may want to move these messages to V(4) or greater

Reply:
This is good idea and I'm concern that we'll neglect it in the future, so I already changed that in all the instances that you mentioned. Thanks!